docs/doxygen-user/multi-user/installPostgres.dox

```diff
@@ -0,0 +1,119 @@
+
+2. Run the installer. The name will be similar to _postgresql-9.5.3-1-windows-x64.exe_.
+
+3. You may accept defaults for all items except for the password as you work through the wizard. Do not lose the password you enter in. This is the PostgreSQL administrator login password.
+
+4. You do not need to launch the StackBuilder nor acquire any more software from it. Uncheck the option to use StackBuilder and press _Finish_.
+
+
+\section install_post_config Configuration 
+
+1. Create a regular database user account that Autopsy will use. You can do this with either of two methods, graphically, or command line. We cover graphically here.
+	
+
+- Use the pgAdmin III  tool and login with the PostgreSQL administrator login. 
+- Right click on <i>"Login Roles"</i> and select <i>"New Login Role..."</i> as shown below:
+<br><br>
+\image html pgAdmin.PNG
+<br><br>
+- Enter the user name you would like to use in the <i>"Role name"</i> field.
+<br><br>
+\image html newLoginRole.PNG
+<br><br>
+- Enter the password on the <i>"Definition"</i> tab.
+<br><br>
+\image html newPassword.PNG
+<br><br>
+- Check <i>"Can create databases"</i> on the <i>"Role Privileges"</i> tab.
+<br><br>
+\image html newRights.PNG
+<br><br>
+- Click <i>"OK"</i>.
+
+2. Edit  <i>C:\\Program Files\\PostgreSQL\\9.5\\data\\pg_hba.conf</i> to add an entry to allow external computers to connect via the network. 
+<br><br>
+First, find your machine's IPv4 address and Subnet Mask (Press _Start_, type _cmd_, type _ipconfig_ and parse the results. The IP address is shown in yellow below.
+<br>
+\image html postgresqlinstall3.PNG
+<br>
+The following is an example rule that allows all clients on the 10.10.192.x  subnet to connect using md5 authentication.
+<br>
+> host&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;all&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;all&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;10.10.192.0/24&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;md5
+<br>
+__Subnet Mask Rules of thumb:__
+    - If your Subnet Mask is 255.255.0.0, your rule should look like this: A.B.0.0/16, where A is the first octet in your IP address and B is the second octet.
+<br>
+    - If your Subnet Mask is 255.255.255.0, your rule should look like this: A.B.C.0/24, where A is the first octet in your IP address, B is the second octet, and C is the third octet. 
+<br><br>
+Add the line highlighted in yellow below, formatted with spaces between the entries, adjusting the IP address to an appropriate value as described above.
+<br><br>
+\image html postgresqlinstall4.PNG
+<br>
+<br><br>
+If you intend to use PostgreSQL from machines on a different subnet, you need an entry in the _pg_hba.conf_ file for each subnet. 
+<br><br>
+
+3. Uncomment the following entires in the configuration file located at <i>C:\\Program Files\\PostgreSQL\\9.5\\data\\postgresql.conf</i> by removing the leading "#", and change their values <i>"off"</i> as shown below.
```

Comment:
References to this location assume that databases are being stored in the default location. If we expand on the PG installation steps above to suggest that the user pick an alternative location for database storage we will need to update references to the location of the configuration files.